src/components/Table.js

```diff
@@ -260,37 +272,64 @@ export default class Table extends PureComponent {
       checkboxRenderer,
       idKey,
       rowClassName,
-      rowRenderer: renderer
+      rowRenderer: renderer,
+      loadMoreLoaderRowCount,
```

Comment:
In my eyes grouping props in an object is a worse stuff because it leads to wasted re-renders since you provide a new object literal each time. This is why they are all namespaced with `loadMore`. We can use `infiniteLoading` instead it's just way longer.

Reply:
Okay, that makes sense. Let's keep this for now.

Another thing, why are we asking for a `loadMoreTotalCount`? We are assuming we will have a finite list. In case of something like a Twitter timeline, there could be tweets tending to infinity. In that case, it might not always be possible to pass a `loadMoreTotalCount`. 

I think in the table, we would be better off just firing the `loadMore` call whenever the scroll reaches bottom, and let the user handle it in their callback.

What say?